CHANGELOG.md

```diff
@@ -1,9 +1,15 @@
 # Change log
 
+
+## 4.4.0 (2020-04-23)
```

Comment:
If we increase the version (to be consistent with the previous changes (and mine of today! 😆 ) then we should increase also the version of the software in the setup